rclcpp/src/rclcpp/utilities.cpp

```diff
@@ -240,9 +241,9 @@ rclcpp::remove_ros_arguments(int argc, char const * const argv[])
   }
 
   std::vector<std::string> return_arguments;
-  return_arguments.resize(nonros_argc);
+  return_arguments.resize(static_cast<uint32_t>(nonros_argc));
```

Comment:
I think this should probably be a `static_cast<size_t>`, given https://en.cppreference.com/w/cpp/container/vector/resize, no?